requests-for-comments/2023-10-27-base3000-launch.md

```diff
@@ -0,0 +1,34 @@
+# Base 3000 & Notebooks launch
+
+We want to launch the ‘first version’ of PostHog 3000 after the next sprint, which means the week starting November 13th (+ 1 week if needed). We have decided to **include Notebooks** in this launch, so that we can have a big launch that combines functionality and design changes.
```

Comment:
What's the rationale for doing both at once? 

From a marketing POV it won't make too much position, but may be slightly clearer to tell users about these things one at a time, rather than together. And I assume from an engineering POV it's going to be easier and more stable to go one step at a time?

Reply:
Notebooks is ready to go and Base 3000 is ready to go in 2 weeks, and since the same team is responsible for it, we just picked the same date, rather than artificially delaying one or announcing two things separately.

If we would launch Notebooks a week or two before PostHog 3000, it would mean all screenshots and demos we produce will be outdated 2 weeks later. If we launch PostHog 3000 first, it means another delay for Notebooks and that feature just needs to go out now, at least in Beta 😅

I think the risk is low from an engineering perspective, Notebooks has been used internally for a while and I don't expect 1,000s of users using it in the first couple of days (though that would be great). The first version of PostHog3000 is a very light change, it is mostly dark mode and a different search, even the navigation will stay similar to what we have now, so I also don't expect a lot of instability here.

Unless you think it makes more sense from a Marketing perspective to announce the 2 things separately, so they both get the hype they deserve, then I could see the following order:

1) Notebooks Beta (light marketing)
2) PostHog 3000 launch
3) Notebooks launch

Reply:
This is my main reasoning for launching them together. Notebooks behave quite differently in 3000 world, especially considering the "side panel". I think it will feel broken to release it and then 2 weeks later pull the rug and completely change the experience. 

We're essentially bundling it together so that we minimise the number of dramatic changes in a row, whilst actually having something powerful to offer in our first 3000 release, rather than just a skin.

Reply:
Related to this, it's probably worth sharing that when I've demoed and accidentally had (admittedly, much older) versions of 3000 up, I got some quick questions confirming that "I'll be able to switch back to the old interface") so am assuming based on that that we should expect existing users to initially reject 3000, just by virtue of the fact that it's a 'new' interface and therefore requires new mental models to use. So I would hope that notebooks are just as functional in posthog 1.basic as in 3000. and that they have enough to offer to this set (or not, and we just use notebooks as a carrot)

Reply:
We've agreed with @joethreepwood we would launch the two things together, so I will resolve this conversation for readability.